spec/controllers/Api/v1/expertises_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  module V1
+    describe ExpertisesController, type: :controller do
+      context "#show" do
+        it "returns the expertise of the user" do 
```

Comment:
We create a user for both of these specs, can we extract that to a `let` statement? Also, it's a little clunky having to type FactoryBot.create every time, I think if we set things up using this https://thoughtbot.github.io/factory_bot/setup/summary.html#rspec, we can get around that.